README.md

```diff
@@ -1,24 +1,29 @@
 # HIPIFY
+HIPIFY is a set of tools to translate CUDA source code into portable [HIP][2]  C++ automatically.
 
-### Tools to translate CUDA source code into portable HIP C++ automatically
-## Table of Contents
+# rocBLAS
+rocBLAS is the AMD library for Basic Linear Algebra Subprograms (BLAS) on the [ROCm platform][1]. It is
+implemented in the [HIP programming language][2] and optimized for AMD GPUs.
 
-<!-- toc -->
+## Documentation
+Information about HIPIFY and other user topics can be found in the
+[HIPIFY documentation][3].
```

Comment:
Unneeded line break.

Reply:
Fixed in latest [47e9926](https://github.com/ROCm-Developer-Tools/HIPIFY/pull/737/commits/47e9926058e7fe40d09fe4b083dff7b95296bcce)